sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -1159,7 +1167,6 @@ public void close() throws IOException {
     @Nullable abstract String getTopic();
     @Nullable abstract Coder<K> getKeyCoder();
     @Nullable abstract Coder<V> getValueCoder();
-    abstract boolean getValueOnly();
```

Comment:
yeah, mentioned about it in the description. This is a bug I noticed when I tried to use the writer. 

`withTopic()` is just a wrapper/builder method. All the existing tests use multiple topics, should I another test with single topic?

Testing sink changes are partially done: The validation code is tested in `KafkaIOTest.testValuesSink()`. This test didn't fail earlier  since we don't use a real KafkaProducer that instantiates the serializers. Let me see if I can manually try to instantiate.